EventsExpress.Core/IServices/IUserService.cs

```diff
@@ -12,19 +12,17 @@ public interface IUserService
     {
         Task Create(UserDto userDto);
 
-        Task Update(UserDto userDto);
-
         Task ChangeAvatar(Guid userId, IFormFile avatar);
 
-        Task EditFavoriteCategories(UserDto userDto, IEnumerable<Category> categories);
+        Task EditFavoriteCategories(IEnumerable<Category> categories);
 
         Task SetAttitude(AttitudeDto attitude);
 
-        UserDto GetById(Guid userId);
+        UserDto GetCurrentUserInfo();
 
-        ProfileDto GetProfileById(Guid userId, Guid fromId);
+        ProfileDto GetProfileById(Guid fromId);
```

Comment:
I guess fromId is a current logged in user  Id, but old value of userId - is a profile we want to load